WFP-VAM/prism-app
# 476 analysis comparison; COUNTRY=myanmar

This fixes issue 476.

How to test the feature:

- Use Myanmar
- Choose Hazard layer "Flood extend", you should be able to:
- Select Exposure option in statistic
- When selected, two new inputs appear to fill the operator and the associated value

Notice: I tried to improve the responsivity of the app when we change the statistic (to max to min for example) because values are available in the front. But there is a lot of thing to change and I am not sure of the impacts. So I think it should be great to create new ticket to globally improve the left menu UX to auto-recalculate statistic table and map when any configuration value in the left menu change.

Screenshot/video of feature:
![Capture d’écran du 2023-06-23 10-28-07](https://github.com/WFP-VAM/prism-app/assets/100352902/302ebc6a-6b8e-4f65-a5ce-cb362e377e50)
